src/content/en/updates/2019/08/get-started-with-gpu-compute-on-the-web.md

```diff
@@ -456,10 +457,12 @@ function on a set of data is called dispatching.
   </figcaption>
 </figure>
 
-In our code, “x” and “y” will be respectively the number of rows of the first
-matrix and the number of columns of the second matrix. With that, we can now
-dispatch a compute call with `passEncoder.dispatch(firstMatrix[0],
-secondMatrix[1])`.
+
+The size of the workgroup grid for our compute shader is `(8, 8)` in our WGSL
+code. Because of that, “x” and “y” that are respectively the number of rows of
```

Comment:
Maybe explain that dispatch argument give the number of workgroup grids to run.

Reply:
Done